db/migrate/20201031201347_add_status_to_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToOrders < ActiveRecord::Migration[5.2]
```

Comment:
Hooray! Nice migration. I was looking through the stories and realized that we'd need to add this column. Great job team!